benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -292,19 +292,19 @@ def _verify_virtctl_vms(self, delay=10):
                                                       odf_version=self._odf_version)
                     # Error log with details of failed VM, for catching all vm errors
                     logger.error(f"Failed to verify virtctl SSH login for the following VMs: {', '.join(failure_vms)}")
-                # Upload artifacts
-                if self._google_drive_shared_drive_id:
-                    self.upload_run_artifacts_to_google_drive()
-                elif self._endpoint_url and not self._google_drive_shared_drive_id:
-                    self.upload_run_artifacts_to_s3()
-                else:
-                    self._save_artifacts_local = True
-                if self._es_host:
-                    self._data_dict.update({'run_artifacts_url': self.get_run_artifacts_google_drive()})
-                    # upload several run results
-                    self._upload_to_elasticsearch(index=self._es_index, kind=self._kind, status=self._status,result=self._data_dict)
-                    # verify that data upload to elastic search according to unique uuid
-                    self._verify_elasticsearch_data_uploaded(index=self._es_index, uuid=self._uuid)
+                    # Upload artifacts
+                    if self._google_drive_shared_drive_id:
+                        self.upload_run_artifacts_to_google_drive()
+                    elif self._endpoint_url and not self._google_drive_shared_drive_id:
```

Comment:
Don't need the "and not self._google_drive_shared_drive_id" because that's already covered above.

Reply:
fixed